docs/site/migration/auth/built-in.md

```diff
@@ -6,8 +6,45 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/migration-auth-built-in.html
 ---
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3719](https://github.com/strongloop/loopback-next/issues/3719)
-" %}
+## Migrate the authentication flow
+
+### Request access tokens via login
+
+1. Implement the following functions
+
+- User service
+
+  - https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/services/user-service.ts
+
+- Token service
+
+  - https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/services/jwt-service.ts
+
+- Login method
+
+  - https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/controllers/user.controller.ts#L204
+
+2. Reuse the `User` database from LB3
+
+- Datasource for the User database
+- UserCredentialsRepository
+
+  - https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/repositories/user-credentials.repository.ts
+
+### Protect API calls with access tokens
+
+- JWT strategy
```

Comment:
AFAIK, LoopBack 3 applications are typically not using JWT strategy, because we don't have first class support for that.

Maybe it's possible to set up JWT auth strategy in a LB3 app using loopback-component-passport? That's out of scope of this part of the migration guide, where we should focus on the built-in auth & auth as provided by LB3 core.